File: BubbleSort.cpp
#include<iostream>

void BubbleSort(long *array, int sizeOfArray);

int main() {
    int sizeOfArray;
    std::cin >> sizeOfArray;

    long *array = new long[sizeOfArray];

    for (int i = 0; i < sizeOfArray; i++)
        std::cin >> array[i];

    BubbleSort(array, sizeOfArray);

    for (int i = 0; i < sizeOfArray; i++)
        std::cout << array[i] << " ";
}

void BubbleSort(long *array, int sizeOfArray) {
    int temp;

    for (int i = 0; i < sizeOfArray - 1; i++)
        for (int j = 0; j < sizeOfArray - 1; j++)
            if (array[j] > array[j + 1]) {
                temp = array[j];
                array[j] = array[j + 1];
                array[j + 1] = temp;
            }
}

File: BubbleSort.py
def BubbleSort(array):
    for i in range(0, len(array)):
        for j in range(0, len(array) - 1):
            if array[j] > array[j + 1]:
                temp = array[j]
                array[j] = array[j + 1]
                array[j + 1] = temp


unsortedArray = [4, -2, 16, 5, 1, 0, 8, 11]

BubbleSort(unsortedArray)

for elem in unsortedArray:
    print(elem, " ", end="")
